docs/development/pigs/pig-024.rst

```diff
@@ -0,0 +1,267 @@
+    - the members of the Coordination Committee, as '(Gammapy Coordination Committee)',
+    - the list of contributors of the associated LTS ordered by alphabetical order, as '(Gammapy Contributors)'
+
+    In this case, a comment on the author list composition should be added. Extracted from the Astropy project,
+    one can precise:
+
+    "The author list has three parts: the authors that made significant contributions to the writing of the paper,
+    the members of the Gammapy Project Coordination Committee, and contributors to the Gammapy Project in alphabetical
+    order. The position in the author list does not correspond to contributions to the Gammapy Project as a whole. A more
+    complete list of contributors to the core package can be found in the package repository and at the Gammapy team
+    webpage."
+
+2. Corresponding author
+    It is the 'Gammapy Coordination Committee', associated with its usual mailing list (GAMMAPY-COORDINATION-L@IN2P3.FR).
+
+3. Acknowledgement
+    One has the freedom to precisely mention acknowledgements associated with the publication. In practice, it is
+    recommended to precise the grants or fellowships given to some authors. One should in any case always acknowledge
+    the Astropy project, to which we are affiliated, and our mandatory external libraries (e.g. numpy, scipy, matplotlib,
+    iminuit).
+
+Contribution in conferences
+-----------------------------
+
+The section is about any contribution in conferences (a talk, a poster and their associated proceedings) related to
+the Gammapy project itself. It does not concern a technical or scientific work that uses Gammapy as an open library.
+In this case, the citation scheme of Gammapy should be used by these authors.
+
+As the length of the author list is generally a constrain, the author list is reduced to the short list of contributors
+for the conference, followed by 'The Gammapy team' associated with a link to the Gammapy team webpage:
+
+ ::
+
+  oo, ff, tt, for `the Gammapy team <https://gammapy.org/team.html>`_
+
+If there is a corresponding author, the 'Gammapy Coordination Committee' associated with its usual mailing list, is
+used. Concerning the acknowledgement, the Astropy project should always be mentioned, and if possible our mandatory
+external libraries.
+
+
+4. Metadata files
+==================================
+
+Depending on the software repository, different metadata files are used in the eco-system of Open Source research
+software.
+
+The file ``codemeta.json`` is used by HAL and Software Heritage, and recommended by
+`ESCAPE <https://escape2020.pages.in2p3.fr/wp3/eossr/v0.3.2/metadata.html>`_ or
+`EOSC <https://www.eosc-pillar.eu/news/illustrating-benefits-eosc-research-communities-germany>`_.
+
+The file ``CITATION.cff`` is used by Zenodo and GitHub. Its format should follow the rules set up by the
```

Comment:
I think there should be as statement, that the `CITATION.cff` will be used as the main file to be maintained, while the `codemeta.json` will be automatically created. The `CITATION.cff` is the preferred maintenance file, because it is human readable.

Reply:
done